web/src/app/components/tasks-editor/tasks-editor.component.ts

```diff
@@ -106,6 +106,12 @@ export class TasksEditorComponent {
 
   multipleChoiceTasks = List<Task>();
 
+  ngOnInit(): void {
+    if (!this.formArray.length) {
```

Comment:
Not sure this will work if an "addLoi" has been already been added. This happens when the user clicks "free form data collection" in the previous step. @rfontanarosa can you confirm?

Reply:
You're right. The logic didn't work for already pre-existing tasks. The flow that happened in the debugger was as follows:

1. ngOnChange -> No tasks exists yet
2. ngOnInit  -> initialize the capture location task. This was a problem because firebase hasn't yet pulled existing tasks.
3. ngOnChange -> firebase finished pulling task data from firestore which caused any existing forms in the formArray to be overwritten; thus, losing the default capture location task.

So I updated the PR to add a capture location task anytime a new job has been created.

Reply:
Thank you for fixing that, and sorry for the delay!